package Stream.parallelStreams;

public class Student {

    String name;
    int score;

    Student(String name,int score){
        this.name=name;
        this.score=score;
    }

    public String getName(){
        return this.name;
    }

    public int getScore(){
        return  this.score;
    }



}
